build.gradle

```diff
@@ -262,7 +262,7 @@ allprojects {
     ext {
         androidxTestVersion = "1.4.0"
         appCompatVersion = "1.2.0"
-        azureCommunicationCommonVersion = "1.0.2"
+        azureCommunicationCommonVersion = "1.0.3"
```

Comment:
revert this back, this version is used by dependent SDKs and should be updated as a post-release update after the upcoming release. it should fix the pipeline failure

Reply:
Fixed. Thank you.